modules/lang-painless/src/main/java/org/elasticsearch/painless/Locals.java

```diff
@@ -37,36 +39,14 @@
  */
 public final class Locals {
 
-    /** Reserved word: params map parameter */
-    public static final String PARAMS = "params";
```

Comment:
All these things are Elasticsearch specific so they reside in `GenericElasticsearchScript` only.